surrounded ZipFile write() method in try/except block which raises a custom Exception

sarabrajsingh opened the pull request:
custom error raised when an invalid symlink is detected

Commit:
surrounded ZipFile write() method in try/except block which raises a custom error when an invalid symlink is detected

jbradberry commented on ansible_runner/utils/streaming.py:
```diff
@@ -17,7 +17,11 @@ def stream_dir(source_directory, stream):
+                        try:
+                            archive.write(os.path.join(dirpath, fname), arcname=os.path.join(relpath, fname))
+                        except FileNotFoundError as e:
+                            e.strerror = "invalid symlink detected"
```
I think it would be better to raise a new exception with its own message than modifying an existing exception.

That said, perhaps we're going about this wrong.  We should probably just do an `os.path.isfile()` or something like that before doing the `archive.write()`.

sarabrajsingh replied:
removed try/except for `if conditional` in latest commit `d9285f6`

Commit:
retired try/except block in favour of a if/else conditional that raises an Exception when an invalid symlink is surfaced.

removed un-used import

fixing 'ansible_runner/utils/streaming.py:10:1: E302 expected 2 blank lines, found 1' thrown by flake8

formatted ansible_runner/utils/streaming.py file with python-black

jbradberry reviewed (commented):
It looks like you've got a couple of problems with whitespace, but I think this is the right idea.

jbradberry commented:
I'm kind of surprised that black didn't automatically fix these.
